### DISCUSSION TITLE
Home page improvements in pre-release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#2683] [Announcements]
Just to note an improvement I've made in latest pre-release builds (https://www.sumatrapdfreader.org/prerelease).

Before on startup Sumatra would show Home page with thumbnails of recently opened documents. Opening a document would replace the home page with the document.

That means that thumbnails were no longer available for quick access.

With the change, the Home page is now available as a first tab.

I've also fixed a quirk that was surprising to some: we wouldn't restore a session with only one file opened. My thinking was: we open on Home tab in this case, which makes it easy to re-open the file.

Turns out it was confusing so now we'll restore the session for a single file as well.

This opens the door for potential improvements. 

Now that Home tab is always visible, it makes sense to make it more functional e.g. increase number of thumbnails shown, make them scrollable etc.

It also opens the door for other kinds of tabs e.g. a tab for browsing a folder, which displays files available in a folder to allow easy opening.

But I don't want to get ahead of myself. For now a persistent Home tab is a nice improvement.

**Comment by LumitoLuma** [comment]
Like it a lot! Just a few very minor suggestions:
- I would change the home tab text to a simple "home icon", and make it more compact.
- I would also make that tab, by default, fixed (impossible to close) and always on the left.

Thanks for this great PDF reader!

**Reply by kjk** [reply]
Good ideas. One step at a time.

**Reply by jddcef** [reply]
Don't make the tab fixed, rather just give tabs an option to "fix or lock or pin", like a browser's tabs.

**Reply by ghaspias** [reply]
I support making Home Tab pinned by default, but letting the user control it (at least from the Advanced Settings) instead of forcing it to the user. Also useful, a command to reopen the Home Tab.

**Reply by GitHubRulesOK** [reply]
@ghaspias 
you can disable hometab see coment below https://github.com/sumatrapdfreader/sumatrapdf/discussions/2683#discussioncomment-2908485
However toggle on and off would need separate program planning / recoding currently its not instant  like some others thus needs a restart like new window or drag all tabs to new window. Dang it I crashed doing that, but it was a problem file I had open !!

**Comment by GitHubRulesOK** [comment]
Interesting but now when I click to open the only last file I opened
I have the background Home tab and every time I click the current file icon @ home it opens yet another additional copy
perhaps better if it re-focused the current main tab, and needed say right click or shift to open anew

**Comment by kjk** [comment]
I've added `NoHomeTab` advanced option to disable always showing Home tab.

**Reply by samepaul** [reply]
Oh thanks god. This "feature" is really annoying.

**Comment by dominiosc** [comment]
For the home tab you can add next or under to "Open a document" the last used paths.
It would also be nice to be able to configure shortcuts to folders or links to open as with alt+o for example for \EBOOKS \ Magazines \BROCHURES \Sheets \Projects ...etc. whatever you like to add in home.

The program group at windows bar not show any name if stay active home tab.

**Reply by kjk** [reply]
Suggestions like this are better as issues. An issue tracks implementation status and ensures the issue is not forgotten.

Suggestions left in discussions will most likely be forgotten as I won't be re-reading discussion comments.

That being said, for access to recently opened documents I recommend using using Command Palette `Ctrl + K` which shows recent files at the top.

**Reply by dominiosc** [reply]
Thanks, I append to an issue. The suggestion is for work with the documents of a dir, but if you are thing to remove alt+o function maybe not is much utility
